.github/workflows/tests-npgsql.yml

```diff
@@ -0,0 +1,73 @@
+name: Npgsql Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test:
+    name: ".NET: ${{ matrix.dotnet-version }}
+     Npgsql: ${{ matrix.npgsql-version }}
+     CrateDB: ${{ matrix.cratedb-version }}
+     on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]  # , macos-latest
+        dotnet-version: ['3.1.x', '5.0.x']  # 6.0.0-preview.4
+        cratedb-version: ['4.3.2', '4.4.3', '4.5.1', 'nightly']
+        npgsql-version: ['5.0.5', '6.0.0-preview4']
+        #include:
+        #- dotnet-version: 3.1.x
+        #  dotnet-framework: netcoreapp3.1
+        #- dotnet-version: 5.0.x
+        #  dotnet-framework: net5.0
```

Comment:
```suggestion
```

Reply:
Thanks for spotting this remnant. It was a leftover from my attempt to make the configuration smarter, see next comment. I've removed it on behalf of fc64076aa.